assets/airports/kdab.json

```diff
@@ -561,7 +561,7 @@
             "destination": "kdab",
             "category": "arrival",
             "route": "FOBIN.Q93.MALET..OMN",
-            "altitude": 23000,
+            "altitude": 6000,
```

Comment:
I think it'd be better to make use of the `commands` key to ensure aircraft enter the TRACON at 6000ft, rather than forcing them to spawn at 6000ft. (See https://github.com/openscope/openscope/blob/develop/documentation/spawnPatternReadme.md)

It may still be necessary to decrease the spawn altitude, but aircraft should ideally spawn at a higher level and then descend to the desired level

Reply:
Testing my change, I still have a plane spawning at 11,000 when I set altitude back to 23,000 and use the `cross MALET A060` command in the file.

Reply:
Try setting the spawn altitude to something like 18,000. If that still doesn't work then I'm happy setting the spawn height to 6k.

Reply:
The first plane that spawns on the `FOBIN.Q93.MALET..OMN` route still spawns at 11,000. All planes past the initial one on that route spawn normally and descend as instructed.

Reply:
Alright, must be something to do with the first spawn logic. We can keep it at 6000 then